docs/source/60_building_apps.md

```diff
@@ -1,39 +1,52 @@
 # Building Downstream Applications {#apps}
 
-ADAM is packaged so that it can be used interatively via the ADAM shell, called from
-the command line interface (CLI), or included as a library when building downstream
-applications.
+ADAM is packaged so that it can be used interactively via the ADAM shell, called
+from the command line interface (CLI), or included as a library when building
+downstream applications.
 
-This document covers three patterns for building applications downstream of ADAM:
+This document covers three patterns for building applications downstream of
+ADAM:
 
 * Extend the ADAM CLI by [adding new commands](#commands)
-* Extend the ADAM CLI by [adding new commands in an external repository](#external-commands)
+* Extend the ADAM CLI by [adding new commands in an external
+repository](#external-commands)
 * Use ADAM as a [library in new applications](#library)
 
 
 ## Extend the ADAM CLI by adding new commands {#commands}
 
 ADAM's CLI is implemented in the adam-cli Apache Maven module of the
 [bdgenomics/adam](https://github.com/bigdatagenomics/adam) repository, one
-.scala source file for each CLI action (e.g. [Transform.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala)
-for the [transform](#transform) action), and a main class ([ADAMMain.scala](https://github.com/bigdatagenomics/adam/blob/master/adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala))
+.scala source file for each CLI action (e.g.
```

Comment:
I think this is fine as is, but if you'd like to make a pass at touching it up, that's a-OK with me.